core-it-suite/src/test/java/org/apache/maven/it/TestSuiteOrdering.java

```diff
@@ -50,6 +50,9 @@ private static void infoProperty( PrintStream info, String property )
     {
         try
         {
+            System.clearProperty( "maven.bootclasspath" );
+            System.clearProperty( "maven.conf" );
+            System.clearProperty( "classworlds.conf" );
```

Comment:
`maven.bootclasspath` and `classworlds.conf` are used by Verifier - can be null, will be discovered according to `maven.home` used by test. 
By the way, should by fixed in Verifier 😄 I have a plan for it ...

`maven.conf` is used by Maven itself ... grrry 
https://github.com/apache/maven/search?l=Java&q=maven.conf

I hope that `DefaultMavenSettingsBuilder` is not used ....